cmd/edit/cluster/cmd.go

```diff
@@ -48,14 +51,41 @@ var args struct {
 	httpsProxy                string
 	noProxySlice              []string
 	additionalTrustBundleFile string
+
+	// Upgrade schedule options
+	upgradeScheduleDay  string
+	upgradeScheduleTime string
+}
+
+var dayOptions = []string{
```

Comment:
Do we really need this an also `daysMap`? seems like a duplication. We can get the map keys (keys := reflect.ValueOf(daysMap).MapKeys())